bridge/client/app/_services/notifications.service.ts

```diff
@@ -18,7 +18,10 @@ export class NotificationsService {
 
   addNotification(type, message) {
     let notification = Notification.fromJSON({ type, message });
-    this._notifications.next([...this._notifications.getValue(), notification]);
+    const duplicateNotifications = this._notifications.getValue().filter(n => n.type === notification.type && n.message === notification.message);
+    if(duplicateNotifications.length === 0) {
+      this._notifications.next([...this._notifications.getValue(), notification]);
+    }
```

Comment:
If I read this code in 1 month, I wouldn't understand why we are doing this here...

Please add some comments here (check if notification already exists), and reference the issue itself for future reference

Reply:
Also, you probably should add a ` ` -between `if` and `(` 😉

Reply:
Added comments for clarification